2024/incoming/programming-like-a-beast.pod

```diff
@@ -0,0 +1,128 @@
+Author: tim.potapov+advent@gmail.com
+Title: Programming like a BEAST
+Topic: e
+
+=encoding utf8
+
+L<https://metacpan.org/pod/e>
+
+=head1 Enhance Your Development Workflow with the e Module:
+
+You know how certain aspects of development can be
+a real drag - slow and pretty boring?
+I’ve definitely been there. But here’s where the e
+module comes in and changes the game.
+
+=head1 Rapid Prototyping:
+
+The e module comes loaded in with many features
+that can aid in rapid prototyping as well as testing.
+
+=head2 Let's Make Selenium Testing Less Painfully Again:
+
+One of its cool features is the C<repl> breakpoint
+command, which can seriously simplify debugging and
+testing processes across the board.
+
+It can also be used from the command line like this:
+
+=begin perl
+
+    perl -Me -e 'my $var = 111; repl'
+
+=end perl
+
+With that simple one-liner, you trigger an
+interactive breakpoint which pauses the execution
+of the program and allows you to investigate data
+and try things out. Once satisfied, you can quit
+the debugger and code flow would continue as usual.
+
+So, instead of dealing with all the usual hassle
+of setting "breakpoints" and debugging manually
+(yes, I taking about you C<print>), you can use
+C<e> to just pop in a breakpoint whenever you
+need it. This means you can pause execution,
+check out what’s going on, and make adjustments
+on the fly. It makes the whole debugging experience
+a lot more interactive and, dare I say, even
+somewhat enjoyable.
+
+A prime example of where C<e> shines is in
+Selenium testing, where it can significantly
+ease the process. Selenium tests can be
+notoriously slow and cumbersome, but its
+C<repl> breakpoint feature makes it less painful.
+However, it’s not just for Selenium, C<e> can
+enhance various development tasks by making
+debugging and testing more efficient (C<time =~ /money/>).
+
+=head2 Performance Efficiency:
+
+Worried about performance? Well, stop it!
+C<e> has been optimized to have a very tiny
+footprint to conserve space and resources.
+Also, it makes great use of Just In Time (JIT)
+module loading: only what you'd use would be
+loaded in. For instance, I use it on my
+smartphone with Termux, and it’s incredibly
+fast - even with the phone's limited resources.
+The impact is practically negligible, and it
+runs smoothly without any noticeable delays.
+
+=head2 User Testimonials:
+
+[Placeholder for user testimonials section]
+
+=head2 Interactive Demos:
+
+To see C<e> in action, check out this L<Perl Banjo|https://perlbanjo.com/dbce20686b>
```

Comment:
As far as I can tell, perlbanjo.com links will expire (possibly in 30 days?). The blog posts in the calendar can live on for decades.

It might be worth considering either a youtube video, screenshots or https://docs.asciinema.org/manual/asciicast/v2/ (which may also not live forever, but I think would have a longer lifetime than perlbanjo.

Reply:
@bessarabov can you confirm how long the perlbanjo.com links remain active?

Reply:
The links created through https://perlbanjo.com/ expire after 30 days by default. This can't be changed through the interface. However, for this particular case, @poti1 and I discussed it, and I manually set the "persistent" flag on the link. This means it will remain active beyond the standard 30-day period. Essentially, it will stay valid as long as the perlbanjo.com project itself is up and running (and I have no plans to retire it).

Reply:
Thanks, @bessarabov! That sounds great to me.